src/app/shared/filesystem/file-list-view/components/folder-list-item/folder-list-item.component.html

```diff
@@ -1,20 +1,22 @@
-<div (click)="handleClick($event)" class="grid-row wrapper">
-  <div class="text-contained inline-flex">
+<div role="link" tabindex="0" attr.aria-labelledby="name description updated" (click)="handleClick($event)" class="grid-row wrapper">
+  <div class="text-contained inline-flex" id="name" attr.aria-label="{{ folder?.getName() }} Folder">
     <i class="fas fa-folder icon"></i>
     {{ folder?.getName() }}
   </div>
-  <div class="text-contained description">
+  <div class="text-contained description" id="description" attr.aria-label="{{ folder?. description }}">
     <span>{{ folder?.description }}</span>
   </div>
-  <div class="text-contained">
+  <div class="text-contained" id="updated" attr.aria-label="Last updated {{ timestampAge }}">
```

Comment:
Since this is a component, the id property here should be more specific since we can't guarantee that there won't be any other elements rendered to the page with the same id.

Reply:
Resolved with 0f43c62